### DISCUSSION TITLE
Micronaut Framework 5 Planned Release Date

### DISCUSSION THREAD

**Discussion by sdelamo** [micronaut-projects/micronaut-core#10570] [Planning]
The core committers teams, Unity Foundation + Oracle Labs, had a Micronaut Framework 5 planning meeting last Thursday. We have moved the scheduled date for Micronaut Framework 5 release to October 21st. 

Apart from getting more time to complete the [Micronaut Framework 5.0 tasks](https://github.com/orgs/micronaut-projects/projects/47/views/1), this updated planned date gives more time for third-party releases, which we would like to include in the release such as [Kotlin 2.0](https://github.com/micronaut-projects/micronaut-core/issues/10485), and [Apache Groovy 5](https://github.com/micronaut-projects/micronaut-core/issues/10486), to happen.

**Comment by vs-dsva** [comment]
2025?

**Comment by dvoina** [comment]
2026?

**Comment by fatekingsama** [comment]
2026？

**Comment by sdelamo** [comment]
we are working towards releasing Micronaut 5 in the first quarter of 2026

**Comment by dvoina** [comment]
Gracias! Feliz año nuevo!

________________________________

**Comment by scprek** [comment]
Is there an estimate of how many of the 110 tasks remaining in "no status" are necessary to get a stable release (maybe excluding bugs found)? https://github.com/orgs/micronaut-projects/projects/47/views/1.

Are we days/weeks/months away? I didn't see anything in discord.
